### PR TITLE
jsSIP reference

### PR DESCRIPTION
**Pull request by roberto-nogueira**
Please check the change to description about jsSIP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # tryit-jssip
-This project provides an webRTC client that can be used to connect to almost any public webRTC Server.
\ No newline at end of file
+This project provides an webRTC client that can be used to connect to almost any public webRTC Server.
+Note that is used a free Javascript library name jsSIP.
```
Comment (robnog) [inline README.md]: There is a typo. Must be "named" instead of "name"

**Review by robnog — CHANGES_REQUESTED** [review]
Please check my comments

File: README.md (hunk 1 above)
Comment (roberto-nogueira) [reply README.md]: Changed to "named". Thanks.

**Review by robnog — APPROVED** [review]
LGTM

**Pull request merged**